### ISSUE TITLE
feature request: create UTFGrid

### ISSUE THREAD

**Issue by tobwen** [jericks/geoc#3]
## tl;dr
It would be nice to generate _UTFGrid_ in _geoc_'s tile mode.

## background
Although it seems that _UTFGrid_ is no longer needed since the introduction of vector tiles, this is not necessarily correct.

There are cases where geospatial data is subject to copyright law that does not allow the publication as a reverse vector data set (_geoc_ is a good example to recover data from vector tiles that is very close to its original shape). In these cases it is necessary to deliver raster data.

_UTFGrid_ allows offering interactivity to the user, like mouseover, in web mapping libraries, like _Leaflet_ or _OpenLayers_ with attributes without exposing the original or simplified geometry.

**Comment by jericks** [comment]
I added a UTFGrid example to the web site:  http://jericks.github.io/geoc/examples/tile_utfgrid.html.  I hope that helps!

**Comment by tobwen** [comment]
Great work. Thanks a lot.
